src/qibolab/dummy.py

```diff
@@ -0,0 +1,122 @@
+                    "start": 0,
+                    "phase": 0,
+                },
+                "MZ": {
+                    "duration": 1000,
+                    "amplitude": 0.0025,
+                    "frequency": 7226500000,
+                    "shape": "Rectangular()",
+                    "type": "ro",
+                    "start": 0,
+                    "phase": 0,
+                },
+            },
+            1: {
+                "RX": {
+                    "duration": 40,
+                    "amplitude": 0.0484,
+                    "frequency": 4855663000,
+                    "shape": "Drag(5, -0.02)",
+                    "type": "qd",
+                    "start": 0,
+                    "phase": 0,
+                },
+                "MZ": {
+                    "duration": 620,
+                    "amplitude": 0.003575,
+                    "frequency": 7453265000,
+                    "shape": "Rectangular()",
+                    "type": "ro",
+                    "start": 0,
+                    "phase": 0,
+                },
+            },
+        },
+        "two_qubit": {
+            "0-1": {
+                "CZ": [
+                    {
+                        "duration": 30,
+                        "amplitude": 0.055,
+                        "shape": "Rectangular()",
+                        "qubit": 0,
+                        "relative_start": 0,
+                        "type": "qf",
+                    },
+                    {"type": "virtual_z", "phase": -1.5707963267948966, "qubit": 1},
+                    {"type": "virtual_z", "phase": -1.5707963267948966, "qubit": 0},
+                ]
+            },
+        },
+    },
+    "characterization": {
+        "single_qubit": {
+            0: {
+                "readout_frequency": 7226500000,
+                "drive_frequency": 0.0,
+                "T1": 0.0,
+                "T2": 0.0,
+                "sweetspot": 0.0,
+                "threshold": 0.0,
+                "iq_angle": 0.0,
+            },
+            1: {
+                "readout_frequency": 7453265000,
+                "drive_frequency": 4855663000,
+                "T1": 0.0,
+                "T2": 0.0,
+                "sweetspot": -0.047,
+                "threshold": 0.00028502261712637096,
+                "iq_angle": 1.283105298787488,
+            },
+        }
+    },
+}
+
+
+def create_dummy():
+    """Create a dummy platform using the dummy instrument."""
+
+    # Create channel objects
+    channels = ChannelMap()
+    channels |= ("readout", "drive")
+    channels |= (f"flux-{i}" for i in range(6))
+
+    # Create dummy controller
+    instrument = DummyInstrument(NAME, 0)
+    # Create platform
+    platform = Platform(NAME, RUNCARD, [instrument], channels)
+
+    # map channels to qubits
+    for qubit in platform.qubits:
+        platform.qubits[qubit].readout = channels["readout"]
+        platform.qubits[qubit].drive = channels["drive"]
```

Comment:
It's a bit weird that all qubits share the same drive channel (and also that there are more flux channels than drive channels), but if the tests are designed to work with this assumption it should be fine. Also the flux channels (6) are more than the number of qubits (2).

Reply:
Yes you are right.
I can differentiate also the drive channel.

Reply:
Done @stavros11